target/linux/ipq806x/files-4.19/arch/arm/boot/dts/qcom-ipq8064.dtsi

```diff
@@ -558,9 +559,9 @@
 			reg = <0x108000 0x1000>;
 			qcom,ipc = <&l2cc 0x8 2>;
 
-			interrupts = <0 19 0>,
-				     <0 21 0>,
-				     <0 22 0>;
+			interrupts = <GIC_SPI 19 IRQ_TYPE_LEVEL_HIGH>,
```

Comment:
You've replaced 0=IRQ_TYPE_NONE with 4=IRQ_TYPE_LEVEL_HIGH

Reply:
I mean, commit description doesn't correspond with the patch content.

Reply:
About this we should really ask @chunkeey why every 0 value has been changed to 4... Restoring them... Checked some other dtsi online and they all have 0 value so... ? Error in conversion ?